resources/content/config-content/haproxy/content/etc/haproxy/haproxy.cfg.ftl

```diff
@@ -39,6 +39,7 @@ frontend ${listener.uuid}_frontend
         <#if (listener.sourceProtocol == "http" || listener.sourceProtocol == "https") && (backend.portSpec.domain != "default" || backend.portSpec.path != "default")>
         <#if backend.portSpec.domain != "default">
         acl ${backend.uuid}_host hdr(host) -i ${backend.portSpec.domain}
+        acl ${backend.uuid}_host hdr(host) -i ${backend.portSpec.domain}:${sourcePort}
```

Comment:
@kaos thanks for pointing out the use case. We need to name this ACL rule differently though because we already have the ACL rule above using the same name (${backend.uuid}_host hdr(host)) + include it to the line evaluating it:

use_backend ${listener.uuid}_${backend.uuid}_backend if <#if backend.portSpec.domain != "default">${backend.uuid}_host</#if> <#if backend.portSpec.path != "default">${backend.uuid}_path</#if>